opentk/opentk
# Start of input mode setup for caps lock

### Purpose of this PR

* This implements a forgotten piece for issue #1809 where the input mode needs to be setup for the caps lock key
* Affects the input system


### How To Test

* Create an override to the KeyDown or KeyUp methods of the GameWindow and verify that the modifier is correct.
